changelog/unreleased/kql-search-query-language.md

```diff
@@ -2,16 +2,31 @@ Enhancement: Keyword Query Language (KQL) search syntax
 
 We've introduced support for [KQL](https://learn.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference) as the default oCIS search query language.
 
-Some examples of a valid KQL query are:
-
-* `Tag`: `tag:golden tag:"silver"`
-* `Filename`: `name:file.txt name:"file.docx"`
-* `Content`: `content:ahab content:"captain aha*"`
+Simple queries:
+
+* `tag:golden tag:"silver"`
+* `name:file.txt name:"file.docx"`
+* `content:ahab content:"captain aha*"`
+
+Date/-range queries
+
+* `Mtime:"2023-09-05T08:42:11.23554+02:00"`
+* `Mtime>"2023-09-05T08:42:11.23554+02:00"`
+* `Mtime>="2023-09-05T08:42:11.23554+02:00"`
+* `Mtime<"2023-09-05T08:42:11.23554+02:00"`
+* `Mtime<="2023-09-05T08:42:11.23554+02:00"`
+* `Mtime:today` - range: start of today till end of today
+* `Mtime:yesterday` - range: start of yesterday till end of yesterday
+* `Mtime:"this week"` - range: start of this week till end of this week
```

Comment:
Please define "start of week" https://www.timeanddate.com/calendar/days/first-day-of-the-week.html

Reply:
best question, but i'm prepared for that 🤣 

at the moment it is `time.Monday` but i prepared the search to be able to be configurable, this is not enabled at that moment could we could in a followup pr

Reply:
Can you add this information here? Then we don't need to check the code next time someone asks :)

Reply:
nope, there will be a follow up pr right after....

Reply:
lazy boy :wink: